cdap-data-fabric/src/main/java/io/cdap/cdap/spi/data/sql/PostgreSqlStructuredTable.java

```diff
@@ -328,7 +333,7 @@ private PreparedStatement prepareMultiScanQuery(Map<String, Set<Field<?>>> keyFi
     // We don't close the statement here because once it is closed, the result set is also closed.
     try {
       PreparedStatement statement = connection.prepareStatement(sql);
-      statement.setFetchSize(SCAN_FETCH_SIZE);
+      statement.setFetchSize(cConf.getInt(SCAN_FETCH_SIZE));
```

Comment:
Many test failures below, looks to be related. Please have them fixed.

Reply:
Is this resolved?

Reply:
The related failures are resolved.